pkg/gossip/gossip.go

```diff
@@ -1175,13 +1163,15 @@ func (g *Gossip) MaxHops() uint32 {
 //
 // This method starts bootstrap loop, gossip server, and client
 // management in separate goroutines and returns.
-func (g *Gossip) Start(advertAddr net.Addr, addresses []util.UnresolvedAddr) {
+func (g *Gossip) Start(
+	advertAddr net.Addr, addresses []util.UnresolvedAddr, rpcContext *rpc.Context,
```

Comment:
Let's update the doc comment to explain why we're passing this in rather than using a struct field.

Reply:
Done